client.go

```diff
@@ -68,25 +73,20 @@ func getDockerClient() DockerClient {
 	return dockerClient
 }
 
-// Run the given command in a container based on the given image. The given
-// image string is just the ID of said image. It returns true if the command
-// was successful, false otherwise.
-func runCommandInContainer(img string, cmd []string) bool {
+// Looks for the specified command inside of a Docker image.
+// The given image string is just the ID of said image.
+// It returns true if the command was successful, false otherwise.
+func checkCommandInImage(img string, cmd string) bool {
 	client := getDockerClient()
```

Comment:
Get the client instance right before the `select` statement, since we don't need it before that and it might return early.